packages/core/src/shared/utilities/workspaceUtils.ts

```diff
@@ -693,7 +693,7 @@ export function formatListing(name: string, fileType: vscode.FileType, fullPath:
  * You can either pass a custom callback or rely on the default `formatListing`.
  *
  * @param dirUri The folder to begin traversing
- * @param maxDepth Maximum depth to descend (0 => just this folder)
+ * @param maxDepth Maximum depth to descend (0 => just this folder, -1 => unlimited depth)
```

Comment:
instead of passing `-1`, can we make this param optional, and if it's undefined it's unlimited?

Reply:
+1